dev-support/docker-compose/files/livy/LICENSE

```diff
@@ -0,0 +1,353 @@
+
```

Comment:
I would actually keep livy stuff outside of this branch - keeping stuff that can fetch livy seems fine to me, but I don't think copy from the other repo is a right thing to do. Writing a script that can git clone from other places seems fine to me, but again, we'll ask Allen about this.

Reply:
Still working on the review... clicked too soon...